source/read/count-documents.txt

```diff
@@ -72,9 +71,10 @@ to modify the count operation. You can chain these method calls together.
 
 To modify the behavior of the count operation, pass the class instance and
 chained method calls as the last argument to the ``countDocuments()`` method. 
-If you don't specify any options, the driver does not customize the count operation.
+If you don't specify any options, the driver executes the count operation
+without any modified behavior.
```

Comment:
What do you think about just leaving these lines out? To me, it's implied that it will behave like a regular ``countDocuments()`` call.